### PR TITLE
Increase logging level for all important actions (create/delete/activate/deactivate/ensure) for networks and endpoints

### PR DESCRIPTION
**Pull request by Soulou**

### CODE REVIEW

File: endpoint/activate.go [hunk 1]
```diff
@@ -3,12 +3,17 @@ package endpoint
...
+	"github.com/pkg/errors"
+
+	"github.com/Scalingo/go-utils/logger"
 	"github.com/Scalingo/sand/api/params"
 	"github.com/Scalingo/sand/api/types"
-	"github.com/pkg/errors"
 )
 
 func (r *repository) Activate(ctx context.Context, n types.Network, endpoint types.Endpoint, params params.EndpointActivate) (types.Endpoint, error) {
+	log := logger.Get(ctx)
```
Comment (EtienneM) [inline endpoint/activate.go:14]: Don't you want to add network and endpoint ID to the logger? Same question on the other file you modified.

File: integrations/docker/driver_network.go [hunk 2]
```diff
@@ -57,11 +66,17 @@ func (p *dockerNetworkPlugin) AllocateNetwork(ctx context.Context, req *network.
 }
 
 func (p *dockerNetworkPlugin) DeleteNetwork(ctx context.Context, req *network.DeleteNetworkRequest) error {
+	log := logger.Get(ctx).WithField("docker_network_id", req.NetworkID)
```
Comment (EtienneM) [inline integrations/docker/driver_network.go:69]:
```suggestion
	log := logger.Get(ctx).WithField("docker_network_id", req.NetworkID)
	ctx = logger.ToCtx(ctx, log)
```

File: integrations/docker/driver_network.go [hunk 3]
```diff
@@ -87,11 +104,15 @@ func (p *dockerNetworkPlugin) FreeNetwork(ctx context.Context, req *network.Free
 }
 
 func (p *dockerNetworkPlugin) CreateEndpoint(ctx context.Context, req *network.CreateEndpointRequest) (*network.CreateEndpointResponse, error) {
+	log := logger.Get(ctx).WithField("docker_network_id", req.NetworkID)
```
Comment (EtienneM) [inline integrations/docker/driver_network.go:107]:
```suggestion
	log := logger.Get(ctx).WithField("docker_network_id", req.NetworkID)
	ctx = logger.ToCtx(ctx, log)
```

**Review by EtienneM — APPROVED** [review]

File: endpoint/activate.go (hunk 1 above)
Comment (Soulou) [reply endpoint/activate.go:14]: It's the entity loading the data which adds it in the logger, so the HTTP handlers or worker

**Pull request merged**